README.md

```diff
@@ -1,12 +1,12 @@
-# vdeveganca_app
+# V de Vegança App for Android and iOS
 
-A new Flutter project.
+This project will simply the use of our database for search of products and ingredients.
```

Comment:
Did you mean `simplify the use` or `will simply use`? 

I'd also add `the` before search: `database for the search of products and ingredients`

Reply:
Agreed. I meant `simplify` 😬  thanks.